_rules/image-accessible-name-descriptive-qt1vmo.md

```diff
@@ -0,0 +1,295 @@
+---
+id: qt1vmo
+name: Image accessible name is descriptive
+rule_type: atomic
+description: |
+  This rule checks that images with an accessible name describe their topic or purpose.
```

Comment:
I still read the subject of "describe" as being "images with an accessible name" and that makes the sentence very weird (at best).

Reply:
I agree with @Jym77 
My suggestion: "This rule checks that an image's topic or purpose is described by its accessible name."
Other suggestion: "This rule checks that the accessible name of an image describes the image's topic or purpose."